### PR TITLE
Update setup.py

### PR DESCRIPTION
**Pull request by nhsiehgit**

### CODE REVIEW

**Comment by nhsiehgit** [comment]
@yurishkuro ?

File: setup.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
     platforms='any',
     install_requires=[
         'protobuf>=3.0.0b2.post2',
-        'opentracing==2.0.0',
+        'opentracing>=2.0.0',
```
Comment (yurishkuro) [inline setup.py]: >=2,<3 is the preferred way

Comment (asottile) [reply setup.py]: protobuf isn't using that approach :S

Comment (nhsiehgit) [reply setup.py]: Cool I'm fine with either
Will update

Comment (yurishkuro) [reply setup.py]: tbh not sure why this lib even needs protobuf, it might be a legacy dep

**Pull request merged**